projects/components/src/tabs/navigable/navigable-tabs.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { NavigableTab } from './navigable-tab';
+
+@Component({
+  selector: 'htc-navigable-tabs',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <htc-navigable-tab-group>
```

Comment:
This component seems like it does nothing but change the api to to the tab group. Is that something we want to do? We discussed it at the time and designed on the previous API.

Reply:
I think this API makes sense for how we use navigable tabs now, but I could instead move this code inside page-header.comp.ts and remove this layer.

Will update PR in a moment.